robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -18,4 +26,26 @@ public static class OperatorConstants {
 
     public static final int kDriverControllerPort = 0;
   }
+
+  public static class CAN_IDS {
+
+    public static final int CORAL_RUNNER_MOTOR = 29;
+  }
+
+  public static class DIGITAL_INPUT {
+
+    public static final int CORAL_RUNNER_BEAM_BREAK_OUTTAKE_ID = 8;
+    public static final int CORAL_RUNNER_BEAM_BREAK_INTAKE_ID = 7;
+  }
+
+  public static class CORAL_RUNNER {
+
+    public static final SparkBaseConfig MOTOR_CONFIG = new SparkMaxConfig()
+      .idleMode(IdleMode.kBrake)
+      .openLoopRampRate(.25)
+      .voltageCompensation(.12)
```

Comment:
Ah yes, make the motor behave that it is always at 0.12 volts. I think you mean `12` here